app/os_settings_strings.grdp

```diff
@@ -1575,9 +1575,21 @@ Press an assigned switch or key to remove assignment.
   <message name="IDS_SETTINGS_ACCOUNT_MANAGER_LIST_DESCRIPTION" desc="List description for additional account List in Account Manager Settings page.">
     You can add your additional accounts to access websites and apps.
   </message>
+  <message name="IDS_SETTINGS_ACCOUNT_MANAGER_LIST_DESCRIPTION_V2" desc="List description for additional account list in Account Manager Settings page.">
+    You can add additional accounts to your <ph name="DEVICE_TYPE">$1<ex>Bravebook</ex></ph> to use with websites and Android apps. You can also control which accounts are used with Android apps.
```

Comment:
This one probably doesn't matter, but overly aggressive substitution here, need to roll back to Chromebook.

Reply:
Thank you, fixed in 19cc0c8f436703ccbd500f16944a4c2089ef2d27.